### ISSUE TITLE
ZNC cannot connect under proxychains when forked into the background

### ISSUE THREAD

**Issue by jbmorris289** [rofl0r/proxychains-ng#330]
Hello!

I recently installed proxychains4 via apt after finding out that it was available. I use proxychains with ZNC to connect to IRC through Tor.
Today, I tried to use proxychains4 with ZNC and ran into problems, but found an interesting temporary fix for them.
When I first ran ZNC under proxychains4, ZNC could not connect to any network at all (stuck on "connecting" with no error). I was suggested by another user to run znc (under proxychains4) in foreground, which mitigated the problem.

A contributor of ZNC said that this problem could be due to the method proxychains uses for preloading.

Could that be the problem, or is it something I'm doing wrong? Thanks!

**Comment by rofl0r** [comment]
yes, it is a known problem that backgrounded/detached processes won't work with the preload method of proxychains. i still don't know what's causing that, so if someone can share some light on this that would be appreciated.
